### PR TITLE
feature / dummy html for table

### PR DESCRIPTION
**Pull request by JulianLeviston**
This PR creates a bunch of dummy HTML and data so we have something to show on the screen and work with for designing, and to see the UI taking place

dummy_stuff

### CODE REVIEW

File: src/app/Dice.elm [hunk 1]
```diff
@@ -17,3 +20,33 @@ emptyCup =
 cupFromIntList : List Int -> Cup
 cupFromIntList =
     identity
+
+
+viewDie : Die -> Html msg
```
Comment (JulianLeviston) [inline src/app/Dice.elm]: Someone can work on die rendering here. Happy to name it something else if preferred.

File: src/app/Dice.elm [hunk 2]
```diff
@@ -17,3 +20,33 @@ emptyCup =
...
+viewDie die =
+    div
+        [ style
```
Comment (JulianLeviston) [inline src/app/Dice.elm]: Probably would be better to use Elm CSS rather than the built in one. Safer and nicer.

File: src/app/Table.elm [hunk 3]
```diff
@@ -1,4 +1,4 @@
-module Table exposing (view, init, initWithPlayersAndCup)
+module Table exposing (view, init, fromPlayersAndCup)
```
Comment (JulianLeviston) [inline src/app/Table.elm]: renamed because `init` is misleading — there's no state (... yet!) :)

File: src/app/Types.elm [hunk 4]
```diff
@@ -8,6 +8,10 @@ type alias PlayerID =
     Int
 
 
+type alias PlayerName =
```
Comment (JulianLeviston) [inline src/app/Types.elm]: Moved here from previous suggestion by joe.

File: src/app/View.elm [hunk 5]
```diff
@@ -20,7 +20,14 @@ root model =
           --            [ text ("FaceValue: " ++ toString model.currentBid.faceValue) ]
           --        , p []
           --            [ text ("Player: " ++ firstPlayer model.players) ]
-        , tableView [] []
+        , tableView
+            2
+            [ Player "Joe" 6
```
Comment (JulianLeviston) [inline src/app/View.elm]: This is just some random data for the moment.

File: src/app/View.elm [hunk 6]
```diff
@@ -34,8 +41,8 @@ firstPlayer players =
             "No Players Found"
 
 
-tableView : List Player -> Dice.Cup -> Html Msg
-tableView players cup =
+tableView : Int -> List Player -> Dice.Cup -> Html Msg
+tableView currentPlayerIndex players cup =
```
Comment (JulianLeviston) [inline src/app/View.elm]: We need some way to tell the `tableView` which of the players is currently having their turn. Unsure if the backend allows for this.

File: src/app/View.elm [hunk 7]
```diff
@@ -47,4 +54,4 @@ tableView players cup =
         playersTuples =
             List.map2 idAndPlayerTo3Tuple ids players
     in
-        Table.view Types.TableAction <| Table.initWithPlayersAndCup playersTuples cup
+        Table.view Types.TableAction <| Table.fromPlayersAndCup playersTuples cup
```
Comment (JulianLeviston) [inline src/app/View.elm]: `Table.view` currently doesn't allow it to be told which one is selected. Will add this later.

**Comment by mordrax** [comment]
Cool, looking better! :shipit:

**Pull request merged**